bioconda_utils/utils.py

```diff
@@ -492,7 +492,7 @@ def get_deps(meta, sec):
         deps = reqs.get(sec)
         if not deps:
             return []
-        return [dep.split()[0] for dep in deps if dep is not None]
+        return [dep.split()[0] for dep in deps if dep]
```

Comment:
`load_meta_fast` does not evaluate the Jinja functions. Hence, `{{ compiler(...) }}` or `{{ pin_compatible(...) }}` may be empty strings here.
Changing
```python
if dep is not None
```
to
```
if dep
```
just prevents raising `IndexError` in for `dep.split()[0]`.
If we want to handle this correctly, we'd have to evaluate the Jinja constructs.

Reply:
This is fine with me.